scoring/tests/test_maximum_extent_scoring.py

```diff
@@ -159,6 +159,69 @@ def test_both_teams_claim_both_territories(self):
             }
         ])
 
+    def test_territory_becoming_unclaimed_after_it_was_claimed(self):
+        self.assertScores({
+            'ABC': 2,
+            'DEF': 2,
+        }, [
+            {
+                'zone': 0,
+                'station_code': 'PN',
+                'time': 4
+            },
+            {
+                'zone': 1,
+                'station_code': 'PN',
+                'time': 5
+            },
+            {
+                'zone': 0,
+                'station_code': 'PN',
+                'time': 6
+            },
+            {
+                'zone': -1,
+                'station_code': 'PN',
+                'time': 7
+            },
+        ])
+
+    def test_unclaimed_territory_with_others_claimed(self):
+        self.assertScores({
+            'ABC': 4,
+            'DEF': 2,
+        }, [
+            {
+                'zone': 0,
+                'station_code': 'PN',
+                'time': 4
+            },
+            {
+                'zone': 1,
+                'station_code': 'PN',
+                'time': 5
+            },
+            {
+                'zone': 0,
+                'station_code': 'PN',
+                'time': 6
+            },
+            {
+                'zone': 0,
+                'station_code': 'EY',
+                'time': 7
+            },
+            {
+                'zone': -1,
+                'station_code': 'PN',
+                'time': 8
+            },
+            {
+                'zone': 1,
+                'station_code': 'SZ',
+                'time': 9
+            },
+        ])
```

Comment:
Looks like we've lost a newline here:
```suggestion
        ])

```